app/views/matches/_client_files.haml

```diff
@@ -12,8 +12,10 @@
         - if required
           *
         - if file.present?
-          - label = "#{tag} (#{file.effective_date})"
-          = link_to tag, Config.get(:warehouse_url) + "/clients/#{client.remote_id}/files/#{file.id}", target: '_blank'
+          - if file.effective_date.present?
+            - date = " (#{date_format(file.effective_date)})"
```

Comment:
Do you think there's any point in including the date uploaded?  I'm not sure which is the most useful date, uploaded, effective, expiration.